pkg/apis/servicecatalog/v1alpha1/types.go

```diff
@@ -328,6 +340,14 @@ type InstanceSpec struct {
 	//
 	// Immutable.
 	ExternalID string `json:"externalID"`
+
+	// Currently, this field is ALPHA: it may change or disappear at any time
+	// and its data will not be migrated.
+	//
+	// AlphaUser contains information about the user that created this
+	// instance. This field is set by the API server and not settable by the
+	// end-user. User-provided values for this field are not saved.
+	AlphaUser AlphaUserInfo `json:"alphaUserInfo,omitempty"`
```

Comment:
I would strongly recommend against naming the field with an alpha name. Doing so requires breaking all users of the field when the field progresses to stable status. See discussion in https://github.com/kubernetes/community/issues/869